record-layer_test.go

```diff
@@ -131,15 +129,15 @@ func TestDecryptRecord(t *testing.T) {
 	ciphertext2 := unhex(ciphertext2Hex)
 
 	// Test successful decrypt
-	r := NewRecordLayerTLS(bytes.NewBuffer(ciphertext1))
+	r := NewRecordLayerTLS(bytes.NewBuffer(ciphertext1), directionRead)
```

Comment:
This file might benefit from a `newRecordLayerFromBytes`, as in `handshake-layer_test.go`